### DISCUSSION TITLE
our own DOIs?

### DISCUSSION THREAD

**Discussion by psmedia** [knowledgefutures/pubpub#3161] [Support] [answered]
Is there any way that we can use our own DOIs instead of the automatically assigned ones from KF/PubPub? I couldn't identify one. I know that assigning DOIs is a useful service in most use cases, but my experience with aggregators suggests that publishers should control DOIs for their material whereever possible, ie if you find you have to migrate from one platform or URL or another... er as might be a necessity for some users as we figure out how to deal with the (exciting!) future PubPub.

I guess the alternative would be to have pubpub assign the DOIs, and then work with KF and CrossRef to transfer them to us as the publisher?

**Comment by gabestein** [comment, marked as answer]
hello! Yup, we can add your Crossref account to allow you to deposit via your account and customize the suffix. Please send an email to help@pubpub.org requesting it. We can also help you request a transfer of any existing DOIs to your account.

**Comment by viraldg** [comment]
I’ve shared this with a friend who I think would really appreciate it too. Great work! [smm reseller panel](https://viraldg.com/)

**Comment by viraldg** [comment]
That’s an excellent way to look at it. <a href="[https://viraldg.com/"](https://viraldg.com/%22)> cheap smm panel</a>

**Comment by Emma12-ui** [comment]
This is a great point about DOI ownership—it's something more publishers should be thinking proactively about, especially given the growing need for long-term control over metadata and link persistence. In my experience, retaining control over DOIs is crucial when content needs to migrate across platforms or when publishing strategies evolve. It’s good to see that PubPub is offering services around DOI assignment, but ideally, there should be flexibility for publishers who already have DOI registration workflows or affiliations with CrossRef.

Your suggestion to have DOIs initially assigned by PubPub and then transferred to the publisher’s CrossRef account sounds like a reasonable workaround, though perhaps a bit administrative, it ensures continuity and future-proofing. Hopefully, the platform will continue to evolve with these use cases in mind.

Also, for those interested in how different digital platforms handle DOI persistence and control, especially in the context of modding communities and user-generated content, I've shared some of my [game](https://frlegendsapkmod.com/) notes and examples here: https://frlegendsapkmod.com/. It’s a niche angle but might offer some interesting parallels for people managing dynamic content ecosystems.

**Comment by Emma12-ui** [comment]
You're right—currently, PubPub assigns DOIs automatically through the Knowledge Futures (KF) relationship with CrossRef, and there’s no built-in option to use your own DOIs directly. However, a possible workaround is to have PubPub assign the DOI initially, and then work with KF and CrossRef to transfer ownership of those DOIs to your own CrossRef account later. It's not seamless, but it can give you more control in the long run.
